setup.py

```diff
@@ -4,7 +4,7 @@
     name='clearblade',
     packages=['clearblade'],
     install_requires=['requests', 'paho-mqtt>=1.3.0'],
-    version='2.4.4',
+    version='2.4.5',
     description='A Python SDK for interacting with the ClearBlade Platform.',
     url='https://github.com/ClearBlade/ClearBlade-Python-SDK',
     download_url='https://github.com/ClearBlade/ClearBlade-Python-SDK/archive/v2.4.4.tar.gz',
```

Comment:
@savfaire unfortunately in this old file we were used to changing the version in two places. Where you did and before the .tar.gz. Maybe this is a good time to make this smarter and use the version to build the path to the tar.gz. Can you go ahead?